### DISCUSSION TITLE
How to use linux command line to run batch of tasks

### DISCUSSION THREAD

**Discussion by wxystudio** [APSIMInitiative/ApsimX#8790] [General]
I know how to run APSIM on windows with command line https://apsimnextgeneration.netlify.app/usage/commandlineusage/

But is there a guide for linux command line? Thanks

**Comment by byzheng** [comment]
If you already compiled `Models` following the instruction here: https://apsimnextgeneration.netlify.app/development/software/compile-apsimng/, you can call APSIM NG with similar way `Models--apply <configFileName.txt>`.

**Reply by wxystudio** [reply]
Thanks! I'll try it.
